### PR TITLE
Feature se 2322 fx option spotladder

### PR DESCRIPTION
**Pull request by andrewjlp**
# Pull Request Checklist

- [ ] Changes follow the [style guide](https://github.com/finbourne/sample-notebooks/blob/master/docs/FINBOURNE%20notebook%20style%20guide.ipynb)
- [ ] Tests pass
- [ ] Raised the PR against the `develop` branch

# Description of the PR
https://finbourne.atlassian.net/browse/SE-2322

FX Vanilla Spot Ladder: The sheet books and values 2 vanilla options, one call, one put and demonstrates how you can create spot ladders from the valuation data showing PV and Delta across different underlying spot rates

Describe the code changes for the reviewers, explain the solution you have provided and how it fixes the issue

### CODE REVIEW

File: examples/use-cases/instruments/FX Option Vanilla Spotladder.ipynb [hunk 1]
```diff
@@ -0,0 +1,1771 @@
+{
```
Comment (cscialino) [inline examples/use-cases/instruments/FX Option Vanilla Spotladder.ipynb:1]: <p>Nice one. I wonder if we could include a vol smile somewhere in this notebook, though would probably require a bit of work as you'd have to move the strike. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/511/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Comment (cscialino) [inline examples/use-cases/instruments/FX Option Vanilla Spotladder.ipynb:1]: <p>Very nice. Though I think y-axis label is wrong, should be delta?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/511/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Comment (andrewjlp) [reply examples/use-cases/instruments/FX Option Vanilla Spotladder.ipynb:1]: Agree, fixed now

Comment (andrewjlp) [reply examples/use-cases/instruments/FX Option Vanilla Spotladder.ipynb:1]: Issues with the analytics - this has been raised: https://finbourne.atlassian.net/browse/ANA-1666
I will come back and implement a .1Vol report later

**Comment by andrewjlp** [comment]
Completed the changes for all the review comments

**Comment by andrewjlp** [comment]
Completed the changes for all the review comments

**Comment by andrewjlp** [comment]
Build checks are not running.

**Pull request closed without merging**